frontend/package.json

```diff
@@ -37,9 +37,9 @@
     "@emotion/react": "^11.10.5",
     "@emotion/serialize": "^1.1.1",
     "@emotion/styled": "^11.10.5",
-    "@glideapps/glide-data-grid": "5.2.1",
-    "@glideapps/glide-data-grid-cells": "5.2.1",
-    "@glideapps/glide-data-grid-source": "5.2.1",
+    "@glideapps/glide-data-grid": "5.2.2-beta2",
```

Comment:
What features do we get from 5.2.2-beta2?

Reply:
It fixes a couple of small issues: e.g. currently, the UI crashes if you copy a table with empty list values. But it also adds a couple of new features, e.g. that we can limit the number of decimals or disallow negatives in number input.